cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -367,6 +367,22 @@ public Set<NodeRef> nodes() {
         return nodes;
     }
 
+    /**
+     * Generates list of references to Kafka nodes going to be removed from the Kafka cluster. The references contain both the pod name and
+     * the ID of the Kafka node.
+     *
+     * @return  Set of Kafka node references which are going to be removed
+     */
+    public Set<Integer> removedNodes() {
+        Set<Integer> nodes = new LinkedHashSet<>();
+
+        for (KafkaPool pool : nodePools)    {
+            nodes.addAll(pool.idAssignment.toBeRemoved());
```

Comment:
Maybe having some method in the pool to get the scale -down nodes would be better then directly access the assignment?